packages/salesforcedx-vscode-core/package.json

```diff
@@ -895,11 +895,11 @@
     "jsonValidation": [
       {
         "fileMatch": "sfdx-project.json",
-        "url": "https://forcedotcom.github.io/schemas/sfdx-project.json/sfdx-project.schema.json"
```

Comment:
This should be changed to use the npm module being published. We have something on our roadmap to do it or you can do it in this PR

Reply:
@lcampos Thomas did bring that up when I talked to him about this, and it's straightforward enough. It does have a side affect of not immediately picking up changes to the schema files in the schema repo until there's been a publish there and here, but does have the advantage of reliable delivery schedule and offline access.
Is that repo's module published on a schedule or on each PR? Or is it manual? Or is that still in the works? I don't see any circleci or github actions config in there, and it was last published in May which means the module is missing a couple of the most recent changes, which would make switching to it now have regressions.

Reply:
Yeah, we're aware of those changes when it comes to using a module, this is still something we want to do so the experience across tools is consistent (CLI, VSCode, etc).

Specific to releases we've been doing them manually since we just started moving to that new model. We'll be publishing it again in the next days since we have the v50 updates coming up. That release will be done using CircleCI. After that I think we'll only publish it on every major platform release (spring, summer, winter) or when there are significant changes that must go out.

Reply:
Ok, that makes sense. I should have some time to see about switching this over to the npm module, and we can wait to merge until a newer module is published.